### ISSUE TITLE
Moto E5 Plus No adaptative brightness

### ISSUE THREAD

**Issue by OliverArenas** [phhusson/treble_experimentations#302]
The adaptive brightness function does not appear, the brightness control bar works
PE October

**Comment by phhusson** [comment]
You need to make an overlay with automatic backlight values
There is a guide at

https://forum.xda-developers.com/project-treble/trebleenabled-device-discussion/guide-how-to-make-gsis-overlay-file-t3878974

Le jeu. 31 janv. 2019 à 18:26, T. Malfatti <notifications@github.com> a
écrit :

**Comment by malfatti** [comment]
Oh great, thanks!

**Comment by phhusson** [comment]
Did you end up doing your own overlay?
If so, could you share/PR it to https://github.com/phhusson/vendor_hardware_overlay/ ?

**Comment by malfatti** [comment]
No, I still couldn't get my hands on the stock system, and I didn't have time to try to mount my backup images :) But as soon as I make it, I'll definitely PR back.

Cheers :)

**Comment by phhusson** [comment]
Feel free to ask for help on @phhtreble on telegram

Le mer. 22 mai 2019 à 04:54, T. Malfatti <notifications@github.com> a
écrit :
